mod/contacts.php

```diff
@@ -940,6 +948,13 @@ function _contact_detail_for_template($rr)
 		$sparkle = '';
 	}
 
+	if ($rr['self']) {
+		$dir_icon = 'images/larrow.gif';
+		$alt_text = L10n::t('This is you');
```

Comment:
What a relief, we already use this string :-D

Reply:
I was surprised as well :grimacing: